core/domain/param_domain.py

```diff
@@ -26,19 +26,49 @@
 
 class ParamSpec(object):
     """Value object for an exploration parameter specification."""
+
     def __init__(self, obj_type):
+        """Initializes a ParamSpec object with the specified object type
+
+        Args:
+            obj_type: unicode. The object type with which the parameter is
+            initialized.
+        """
         self.obj_type = obj_type
 
+
     def to_dict(self):
+        """Adds the object type to a dict, with the key as `obj_type`
```

Comment:
This talks too much about the precise implementation and less about what's going on conceptually. Instead, maybe something like:

    Returns a dict representation of this ParamSpec.